intellij-ext/src/main/kotlin/com/metalbear/mirrord/MirrordDefaultConfig.kt

```diff
@@ -19,6 +19,7 @@ data class MirrordDefaultConfig(
     val rustLog: LogLevel = LogLevel.INFO,
     val overrideEnvVarsExclude: String = "",
     val overrideEnvVarsInclude: String = "*",
+    val ignorePorts: String = "",
```

Comment:
Considering how lacking the documentation is around this, will the user even know which ports to ignore? Maybe we should do a default range for intellij?

Reply:
sure, but it is kind of random. we can keep it from 45K-65K wdyt?